promql/engine.go

```diff
@@ -1386,6 +1394,8 @@ func (ev *evaluator) eval(expr parser.Expr) (parser.Value, annotations.Annotatio
 			if ok {
 				return ev.rangeEvalTimestampFunctionOverVectorSelector(vs, call, e)
 			}
+		} else if (e.Func.Name == "sort_by_label" || e.Func.Name == "sort_by_label_desc") && !ev.enableSortByLabel {
+			ev.error(fmt.Errorf("sort_by_label and sort_by_label_desc feature flag not enabled"))
```

Comment:
This makes it sound as if `sort_by_label` and `sort_by_label_desc` are the feature flag names, how about something like this:

```promql
sort_by_label() and sort_by_label_desc() require the "promql-sort-by-label" feature flag to be set
```